app/forms/dependant_details_form.rb

```diff
@@ -0,0 +1,11 @@
+class DependantDetailsForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  DEPENDANT_ATTRIBUTES = %i[adult_dependants child_dependants].freeze
+
+  DEPENDANT_ATTRIBUTES.each do |attr|
+    attribute attr, :integer
+    validates attr, presence: true
```

Comment:
I think we should have a check for integer here?

Reply:
@starswan because we've defined these attributes as integers, any non-integer that gets assigned to them gets immediately cast to an integer _before_ it's validated. So when the validation is run, even if you passed in `3.7` or "forty-three", the check for integer will pass, as the cast value (`3` and `0` respectively in the examples given) _is_ an integer.